Packages/com.unity.inputsystem/InputSystem/Plugins/InputForUI/InputSystemProvider.cs

```diff
@@ -642,16 +641,12 @@ void UnregisterActions(Configuration cfg)
             // The Next/Previous action is not part of the input actions asset
             UnregisterNextPreviousAction();
 
-#if UNITY_EDITOR
-            UnityEngine.Object.DestroyImmediate(m_InputActionAsset);
-#else
-            UnityEngine.Object.Destroy(m_InputActionAsset);
-#endif
+            UnityEngine.Object.Destroy(m_InputActionAsset); // TODO check if this is ok
```

Comment:
This is reverting a change in `develop` and re-adding the TODO.
I assume it's just down to a git merge issue.